clang/docs/LanguageExtensions.rst

```diff
@@ -1187,6 +1187,50 @@ specifier for ``_Float16``, and (unlike ``float``) it will not be implicitly pro
 ``double`` when passed to ``printf``, so the programmer must explicitly cast it to
 ``double`` before using it with an ``%f`` or similar specifier.
 
+Pragmas
+=======
+
+#pragma export
+--------------
+
+Clang supports the export pragma used to indicate an
+external symbol is to be exported from the shared library being built.  The
+syntax for the pragma is:
+
+.. code-block:: c++
+
+  #pragma export (name)
+
+where ``name`` is the name of the external function or variable to be
+exported.  The symbol needs to have external linkage.  The pragma may appear
```

Comment:
Ah, see it now, disregard questions on Function vs var.  The name of the `LookupExternC` should probably be renamed as it implies function, but you really mean 'or global variable'.